CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+1.3.0
+-----
+
+* **2015-03-14** Added support for a simplistic Etag header. This is preferred
```

Comment:
Let's use ‘simple’ instead of ‘simplistic’ as the later has a rather negative connotation (‘oversimplified’) in English.